dist/documentation/themes/dark.js

```diff
@@ -502,4 +502,9 @@ light : {backgroundBase:{
             name: "light.linkMutedHover",
             hex: "#FFFFFF",
             description: "Used for link hover colour for light profile components on contrasting surface backgrounds"
+          }},
+shadow : {base:{
```

Comment:
🍅 🍅 Will we pluralize this once we add more?

Reply:
I don't see us adding more since all the shadows are using the same base colour, just changes for each theme